src/Controller/AdminOrderController.php

```diff
@@ -175,7 +175,7 @@ public function listAction(Request $request, IntlFormatter $formatter, Paginator
         // Paginate the results of the query
         $paginator = $paginator->paginate(
             $list,
-            $request->get('page', 1),
+            (int) $request->get('page', 1),
```

Comment:
We could use the build-in cast from the query request..
```suggestion
            $request->query->getInt('page', 1),
```